### PR TITLE
Doc changes for swift

### PR DESCRIPTION
**Pull request by trooper2013**
Fixed some spacing, missing params  and indentation issues impacting output of jazzy. Added a readme file for  the swift sdk, thats used by jazzy for the main page of docs. Generate docs using the following
```
jazzy --config ./docs/jazzy.yaml  --swift-version 4.0.3 --readme libs/SalesforceSwiftSDK/readme.md
```

### CODE REVIEW

File: docs/jazzy.yaml [hunk 1]
```diff
@@ -6,30 +6,17 @@ exclude:
...
-IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
-FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
-CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
-WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
-WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.'
+  - "**/SwiftExtensions.swift"
+readme: 'libs/SalesforceSwiftSDK/readme.md'
+copyright: '© Copyright 2017–Current salesforce.com, inc. All rights reserved. Salesforce is a registered trademark of salesforce.com, inc.,
```
Comment (bhariharan) [inline docs/jazzy.yaml]: @trooper2013 Maybe we should just remove the `copyright` section here. It's just doc. I know I mentioned adding it in last time around ;-)

File: libs/SalesforceSwiftSDK/readme.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+Mobile SDK 6.1 enhances its Swift offering with PromiseKit adaptations. You can now use promises for these SDK components:
```
Comment (bhariharan) [inline libs/SalesforceSwiftSDK/readme.md]: Probably best not to mention `6.1` here (we already mention this in our `What's New` section. Could you remove `6.1`?

File: docs/jazzy.yaml (hunk 1 above)
Comment (trooper2013) [reply docs/jazzy.yaml]: @bhariharan This copyright shows up in the footer of documentation pages. I think we should keep it.

Comment (bhariharan) [reply docs/jazzy.yaml]: Sounds good.

**Pull request merged**